libs/wallet/src/connectors/vega-connector.ts

```diff
@@ -1,68 +1,93 @@
 import { WalletClientError } from '@vegaprotocol/wallet-client';
-import type * as Schema from '@vegaprotocol/types';
-import type { PeggedReference } from '@vegaprotocol/types';
+import type { LiquidityProvisionSubmission as LiquidityProvisionBody } from '@vegaprotocol/protos/dist/vega/commands/v1/LiquidityProvisionSubmission';
+import type { DelegateSubmission } from '@vegaprotocol/protos/dist/vega/commands/v1/DelegateSubmission';
+import type { UndelegateSubmission } from '@vegaprotocol/protos/dist/vega/commands/v1/UndelegateSubmission';
+import type { OrderSubmission as OriginalOrderPosition } from '@vegaprotocol/protos/dist/vega/commands/v1/OrderSubmission';
+import type { OrderCancellation as OriginalOrderCancellation } from '@vegaprotocol/protos/dist/vega/commands/v1/OrderCancellation';
+import type { OrderAmendment as OriginalOrderAmendment } from '@vegaprotocol/protos/dist/vega/commands/v1/OrderAmendment';
+import type { VoteSubmission } from '@vegaprotocol/protos/dist/vega/commands/v1/VoteSubmission';
+import type { WithdrawSubmission as OriginalWithdrawSubmission } from '@vegaprotocol/protos/dist/vega/commands/v1/WithdrawSubmission';
+import type { ProposalSubmission as OriginalProposalSubmission } from '@vegaprotocol/protos/dist/vega/commands/v1/ProposalSubmission';
+import type { BatchMarketInstructions as OriginalBatchMarketInstructions } from '@vegaprotocol/protos/dist/vega/commands/v1/BatchMarketInstructions';
+import type { Transfer } from '@vegaprotocol/protos/dist/vega/commands/v1/Transfer';
+import type { ProposalTerms } from '@vegaprotocol/protos/dist/vega/ProposalTerms';
+import type { WithdrawExt } from '@vegaprotocol/protos/dist/vega/WithdrawExt';
```

Comment:
Can we export all these types from here like this this way so we don't introduce @vegaprotocol/protos as a dep for other libs

```
import type { vega as vegaProtos } from '@vegaprotocol/protos';

export type OrderSubmission =
  vegaProtos.commands.v1.OrderSubmission.OrderSubmission;
export type OrderCancellation =
  vegaProtos.commands.v1.OrderCancellation.OrderCancellation;
export type LiquidityProvisionSubmission =
  vegaProtos.commands.v1.LiquidityProvisionSubmission.LiquidityProvisionSubmission;
export type DelegateSubmission =
  vegaProtos.commands.v1.DelegateSubmission.DelegateSubmission;
export type UndelegateSubmission =
  vegaProtos.commands.v1.UndelegateSubmission.UndelegateSubmission;
export type OrderAmendment =
  vegaProtos.commands.v1.OrderAmendment.OrderAmendment;
export type VoteSubmission =
  vegaProtos.commands.v1.VoteSubmission.VoteSubmission;
export type WithdrawSubmission =
  vegaProtos.commands.v1.WithdrawSubmission.WithdrawSubmission;
export type ProposalSubmission =
  vegaProtos.commands.v1.ProposalSubmission.ProposalSubmission;
export type BatchMarketInstructionSubmission =
  vegaProtos.commands.v1.BatchMarketInstructions.BatchMarketInstructions;
export type TransferSubmission = vegaProtos.commands.v1.Transfer.Transfer;
```

Reply:
done